### PR TITLE
Update 2.3 readme

### PR DESCRIPTION
**Pull request by lsy323**
As @will-cromar mentioned in the release doc, the README will be shown on the PyPI page, so update README before the final build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,6 +245,7 @@ This is only required on Cloud TPU VMs.
 | Version | Cloud TPU VMs Docker |
 | --- | ----------- |
 | 2.3 | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.3.0_3.10_tpuvm` |
+| 2.2 | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.2.0_3.10_tpuvm` |
```
Comment (JackCaoG) [inline README.md:248]: can we make sure these docker actually exists before merging?

**Review by JackCaoG — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lsy323) [reply README.md:248]: Cannot verify the 2.3 since it's not built yet, however, we need to update the readme before the final build. See title for reason.

Comment (lsy323) [reply README.md]: @will-cromar Thanks for the heads up, just removed

**Comment by lsy323** [comment]
Updated the following:
1. Remove links for whl and docker image, as the page will be on PyPI
2. Remove the kaggle notebook links, point tutorials to GH main page.

**Pull request merged**